demo/admin/package.json

```diff
@@ -13,7 +13,7 @@
         "intl:compile:comet": "formatjs compile-folder --format simple --ast lang/comet-lang lang-compiled/comet-lang",
         "intl:compile:comet-demo": "formatjs compile-folder --format simple --ast lang/comet-demo-lang lang-compiled/comet-demo-lang",
         "intl:extract": "formatjs extract \"src/**/*.ts*\" --ignore ./**.d.ts --out-file lang-extracted/en.json --format simple",
-        "intl:update": "git --git-dir ./lang/comet-lang/.git pull && git --git-dir ./lang/comet-demo-lang/.git pull",
+        "intl:update": "cd ./lang/comet-lang && ls -lah && git pull && cd ../comet-demo-lang && ls -lah && git pull",
```

Comment:
Do we need the `ls` commands?

Reply:
of course not, I was debugging the failing build